ext/curl/curl.stub.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/** @param resource $handle */
+function curl_close($handle): void {}
+
+/**
+ * @param resource $handle
+ *
+ * @return resource|false
+ */
+function curl_copy_handle($handle) {}
+
+/** @param resource $handle */
+function curl_errno($handle): int {}
+
+/** @param resource $handle */
+function curl_error($handle): string {}
+
+/**
+ * @param resource $handle
+ *
+ * @return string|false
+ */
+function curl_escape($handle, string $string) {}
+
+/**
+ * @param resource $handle
+ *
+ * @return mixed|false
+ */
+function curl_exec($handle) {}
+
+/** @return CURLFile */
+function curl_file_create(
+    string $filename,
+    string $mimetype = UNKNOWN,
+    string $postname = UNKNOWN
+) {}
+
+function curl_file_name(string $name): void {}
+
+/**
+ * @param resource $handle
+ *
+ * @return mixed|mixed[]|false
+ */
+function curl_getinfo($handle, int $option = UNKNOWN) {}
+
+/**
+ * @param resource $handle
+ *
+ * @return resource|false
+ */
+function curl_init($handle) {}
+
+/**
+ * @param resource $multi_handle
+ * @param resource $handle
+ */
+function curl_multi_add_handle($multi_handle, $handle): int {}
+
+/** @param resource $multi_handle */
+function curl_multi_close($multi_handle): void {}
+
+/**
+ * @param resource $multi_handle
+ *
+ * @return int|false
+ */
+function curl_multi_errno($multi_handle) {}
+
+/** @param resource $multi_handle */
+function curl_multi_exec($multi_handle, int &$still_running): int {}
```

Comment:
```suggestion
function curl_multi_exec($multi_handle, &$still_running): int {}
```

This is an output parameter, so it should have no input type.